pennylane_lightning/core/src/simulators/lightning_gpu/StateVectorCudaMPI.hpp

```diff
@@ -280,6 +278,46 @@ class StateVectorCudaMPI final
         mpi_manager_.Barrier();
     }
 
+    /**
+     * @brief Prepare a single computational basis state.
+     *
+     * @param state Binary number representing the index
+     * @param wires Wires.
+     * @param use_async Use an asynchronous memory copy.
+     */
+    void setBasisState(const std::vector<std::size_t> &state,
+                       const std::vector<std::size_t> &wires,
+                       const bool use_async) {
+        // This is not functional yet.
```

Comment:
This function doesn't work?

Reply:
I forgot to remove the comment. Thank you, I will remove it.